source/ns_cmdline.c

```diff
@@ -54,19 +54,32 @@
 //#define TRACE_DEEP
 //#define TRACE_PRINTF
 
-
 #ifdef TRACE_PRINTF
 #undef tr_debug
 #define tr_debug(...) printf( __VA_ARGS__);printf("\r\n")
 #endif
 
+// Disable EXCLUDE_TRACE_DEFINES macro to enable trace prints
+#define EXCLUDE_TRACE_DEFINES		1
+#ifdef EXCLUDE_TRACE_DEFINES
```

Comment:
Need a more descriptive flag name. For example MBED_CLIENT_CLI_TRACE_ENABLE.
For example:
```
#ifndef MBED_CLIENT_CLI_TRACE_ENABLE
#undef tr_error
#define tr_error(...)
...
#endif
```